cosmic_ray/operators/zero_iteration_loop.py

```diff
@@ -0,0 +1,18 @@
+import ast
+
+from .operator import Operator
+
+
+class ZeroIterationLoop(Operator):
+
+    """An operator that modifies numeric constants."""
+
+    def visit_For(self, node):  # noqa
+        return self.visit_mutation_site(node)
+
+    def mutate(self, node, _):
+        """Modify the For loop to evaluate to None"""
+        empty_list_node = ast.List(elts=[], ctx=ast.Load)
```

Comment:
I think this needs to be "ctx=ast.Load()", i.e. an instance of `Load`, not just the class object.